src/components/dashboard/components/dictionary/common/ReleaseVersionModal.jsx

```diff
@@ -1,23 +1,21 @@
 import React from 'react';
-import {
-  Button, Modal, ModalHeader, ModalBody, ModalFooter, FormGroup, Input,
-} from 'reactstrap';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter, FormGroup, Input } from 'reactstrap';
 import PropTypes from 'prop-types';
 
 const ReleaseVersionModal = (props) => {
   const {
-    show, click, handleCreateVersion, handleChange, versionId, versionDescription, inputLength,
+    show,
+    click,
+    handleCreateVersion,
+    handleChange,
+    versionId,
+    versionDescription,
+    inputLength,
   } = props;
   return (
     <div className="modal-container">
-      <Modal
-        isOpen={show}
-        onClosed={click}
-        centered
```

Comment:
Is this a formatting change?

Reply:
This are the changes I made when resolving the conflicts.